edi_sale_utm_oca/tests/test_sale_order.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo.tests import tagged
+
+from odoo.addons.base.tests.common import BaseCommon
+from odoo.addons.edi_oca.tests.common import EDIBackendTestMixin
+
+
+@tagged("post_install", "-at_install")
+class TestSaleOrder(BaseCommon, EDIBackendTestMixin):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._setup_records()
+
+        cls.sale_order_model = cls.env["sale.order"]
+
+    def test_source_edi(self):
+        sale_order = self.sale_order_model.create(
+            {
+                "name": "",
+                "origin_exchange_type_id": self.exchange_type_in.id,
+                "partner_id": self.partner.id,
+            }
+        )
+
+        self.assertEqual(
+            sale_order.source_id.id,
+            self.env.ref("edi_sale_utm_oca.utm_source_edi").id,
+        )
```

Comment:
reuse the test in `edi_sale_oca` instead

You can have a look at `edi_sale_ubl_oca` (v14, `edi/pull/759`) to get the idea to test